edalize/rivierapro.py

```diff
@@ -84,7 +90,20 @@ def _write_build_rtl_tcl_file(self, tcl_main):
                 args += ['-quiet']
                 args += ['-work', f.logical_name]
                 args += [f.name.replace('\\','/')]
-                tcl_build_rtl.write("{} {}\n".format(cmd, ' '.join(args)))
+                common_compilation += [f.name.replace('\\','/'),'\\\n']
+
+                if (self.tool_options.get('compilation_mode'))==['sep']:
+                    tcl_build_rtl.write("{} {}\n".format(cmd, ' '.join(args)))
+
+        if (self.tool_options.get('compilation_mode')==['common']) or (self.tool_options.get('compilation_mode')==None):
+            tcl_build_rtl.write("{} \n".format(' '.join(common_compilation)))
+			
```

Comment:
I would prefer the default mode to be `sep` to avoid risk breaking people's code if they rely on the old behaviour.